docs/source/terms_and_model.rst

```diff
@@ -371,9 +371,6 @@ A contiguous region specified by named features.
   locations respectively. If `start` and `end` are on the same arm,
   `start` SHOULD be the more centromeric position (i.e., with lower
   band and sub-band numbers).
-* NamedIntervals are currently unoriented. Future versions of VRS may
-  introduce conventions or attributes that permit explicit
-  orientation.
```

Comment:
Removing this, as it is unclear what is unoriented about this interval class, or how this differs from a SimpleInterval, which is not explicitly unoriented.

Reply:
The goal here was two-fold: 1) dissuade users from creating meaning where none was intended. For example, writing start→end order as telomereic→centromeric and wanting that to imply something. 2) Leave open the possibility that it would be used later.

I'd like to keep some version of this idea. Feel free to propose other wording.

Reply:
reconsidered. Left as-is.